### PR TITLE
chore: Update pull-request-template.md

### PR DESCRIPTION
**Pull request by Disper**
**Description**

Changes proposed in this pull request:

- Updates the PR template to mitigate findings connected to internal audit on microdeliveries

**Related issue(s)**
- #backlog/5699

**Confirm that everything was done**
- [x] PR title starts with prefix
    - feat:     A new feature
    - fix:      A bug fix
    - docs:     Documentation only changes
    - refactor: A code change that neither fixes a bug nor adds a feature
    - test:     Adding (missing) tests
    - chore:    Maintain. Changes to the build process or auxiliary tools/libraries/workflows etc.
- [x] Related issues are linked
- [x] It's clear why is this PR was created and what it does
- [x] All necessary steps are delivered (like e.g. tests, documentation, merging)

### CODE REVIEW

**Review by tobiscr — APPROVED** [review]

File: .github/pull-request-template.md [hunk 1]
```diff
@@ -17,3 +17,8 @@ Changes proposed in this pull request:
 
 **Related issue(s)**
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+**Confirm that everything was done**
+- [ ] Related issues are linked
```
Comment (tobiscr) [inline .github/pull-request-template.md:22]:
```suggestion
- [ ] PR title starts with prefix
    - feat:     A new feature
    - fix:      A bug fix
    - docs:     Documentation only changes
    - refactor: A code change that neither fixes a bug nor adds a feature
    - test:     Adding missing tests
    - chore:    Maintain. Changes to the build process or auxiliary tools/libraries/documentation
- [ ] Related issues are linked
```

**Review by tobiscr — CHANGES_REQUESTED** [review]

File: .github/pull-request-template.md (hunk 1 above)
Comment (koala7659) [inline .github/pull-request-template.md:22]: What about issues created in our private repo, should I link them as well?

Comment (Disper) [reply .github/pull-request-template.md:22]: image
I've tried multiple times and I can't apply this suggestion from github. I will apply it manually.

Comment (Disper) [reply .github/pull-request-template.md]: @koala7659  I've made a suggestion about that point in 10ffc1d40e524bc5cc114563f81604dfbfc25a63

**Comment by Disper** [comment]
@zhoujing2022 
Could we not have the prefixes? We don't know if the value added by manually adding those prefixes is worth the effort.

```
- [ ] PR title starts with prefix
    - feat:     A new feature
    - fix:      A bug fix
    - docs:     Documentation only changes
    - refactor: A code change that neither fixes a bug nor adds a feature
    - test:     Adding tests
    - chore:    Maintain. Changes to the build process or auxiliary tools/libraries/workflows etc.
```

**Review by tobiscr — APPROVED** [review]

**Comment by Disper** [comment]
Aligned with @zhoujing2022  that it would be enough to have 
- feature
- bugfix
- other
would be enough.

Prefixes is just the simplest idea of achieving that, we can actually achieve that in any way we want (e.g. using labels)
To consider - we could have an additional label/prefix that will mention **breaking changes**.

**Pull request closed without merging**